core/banking-protocol/src/main/java/de/adorsys/opba/core/protocol/controller/TppBankSearchController.java

```diff
@@ -0,0 +1,80 @@
+package de.adorsys.opba.core.protocol.controller;
+
+import de.adorsys.opba.core.protocol.domain.entity.Bank;
+import de.adorsys.opba.core.protocol.domain.entity.BankProfile;
+import de.adorsys.opba.core.protocol.service.BankService;
+import de.adorsys.opba.tppbankingapi.search.model.BankDescriptor;
+import de.adorsys.opba.tppbankingapi.search.model.BankProfileDescriptor;
+import de.adorsys.opba.tppbankingapi.search.model.BankProfileResponse;
+import de.adorsys.opba.tppbankingapi.search.model.BankSearchResponse;
+import de.adorsys.opba.tppbankingapi.search.resource.TppBankSearchApi;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@RestController
+@RequiredArgsConstructor
+@CrossOrigin(origins = "*") //FIXME move CORS at gateway/load balancer level
+@SuppressWarnings("checkstyle:ParameterNumber")
+public class TppBankSearchController implements TppBankSearchApi {
+    private static final int DEFAULT_START = 0;
```

Comment:
Why not to inject these via `@Value` / annotation?

Reply:
done